### PR TITLE
Added a basic Achievement component

### PR DESCRIPTION
**Pull request by Tanish2000**
Added a basic component for achievements

Screenshot (88)
Screenshot (89)

### CODE REVIEW

**Comment by Tanish2000** [comment]
> Screenshot (88)
> Screenshot (89)

**Comment by ManojSatishkumar** [comment]
Please resolve conflict

**Comment by Tanish2000** [comment]
conflict resolved

File: src/modules/resume/index.js [hunk 1]
```diff
@@ -8,8 +8,13 @@ import Experience from './experience'
 import Education from './education'
 import Projects from './projects'
 import Skillset from './skill-set'
+<<<<<<< HEAD
```
Comment (venkateshpachigulla) [inline src/modules/resume/index.js:11]: please resolve conflicts. you can use vscode git extension or resolve manually.

File: src/modules/resume/achievements/variants/basic/properties.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+              type: propertyTypes.STRING_MULTI_LINE,
+              value: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+              required: true
+            },
+            {
+                name: 'Achievement text',
+                type: propertyTypes.STRING_MULTI_LINE,
+                value: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                required: true
+              },
```
Comment (AKAspanion) [inline src/modules/resume/achievements/variants/basic/properties.js:32]: Format all files correctly.

File: src/modules/resume/achievements/variants/basic/render.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    return (
+      <div style={{
+        ...styles.root,
+        backgroundColor: theme.value.color,
+        color: theme.value.contrast
+      }}>
+        <h1 style={styles.headlineText}>{headlineText.value}</h1>
+        <hr />
+        {<ol>
+          {achievements.value.map((paragraph, id) => {
```
Comment (AKAspanion) [inline src/modules/resume/achievements/variants/basic/render.js:32]: Please see.
https://github.com/ManojSatishkumar/msk-components/blob/master/src/modules/resume/sample/variants/basic/render.js#L26

**Pull request closed without merging**